R/ci.R

```diff
@@ -111,3 +111,27 @@ appveyor_info <- function(base_path = proj_get()) {
 
   list(url = url, img = img)
 }
+
+#' @section `use_gitlab_ci()`:
+#' Adds a basic `.gitlab-ci.yml` to the top-level directory of a package. This is
+#' a configuration file for the [Gitlab CI/CD](https://docs.gitlab.com/ee/ci/) continuous
+#' integration service for Gitlab.
+#' @export
+#' @rdname ci
+use_gitlab_ci <- function(browse = interactive()) {
```

Comment:
I also don't see any use of `browse`.

Reply:
Also holdover from use_travis(). With gitlab, there is no registration for the ci process. As long as .gitlab-ci.yml is present in the root directory of the package it runs by default.

Reply:
Should the `browse` argument go away then?

Reply:
I think for this one it would make sense.